tests/test_ble_interface_core.py

```
@@ -4864,7 +4814,12 @@ def _raise_start_failure(_thread: object) -> None:
         )
 
         with pytest.raises(RuntimeError, match=START_FAILED_MSG):
-            iface._start_receive_thread(name="BLEReceiveStartFailure")
+            if invoke_start == "facade":
+                iface._start_receive_thread(name="BLEReceiveStartFailure")
+            else:
+                BLELifecycleService._start_receive_thread(
+                    iface, name="BLEReceiveStartFailure"
+                )
```

Ruff (0.15.6)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)